src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -598,8 +598,7 @@ private TypeSymbol BindVariableType(CSharpSyntaxNode declarationNode, Diagnostic
             // or it might not; if it is not then we do not want to report an error. If it is, then
             // we want to treat the declaration as an explicitly typed declaration.
 
-            RefKind refKind;
-            TypeSymbol declType = BindType(typeSyntax.SkipRef(out refKind), diagnostics, out isVar, out alias);
```

Comment:
>typeSyntax.SkipRef(out refKind) [](start = 43, length = 31)

Are we certain it is safe to remove this code? There are multiple callers and it doesn't look like all of them are skipping the ref.